.atomist/handlers/event/PullRequest.ts

```diff
@@ -31,23 +31,7 @@ import { PullRequest } from "@atomist/cortex/PullRequest";
 @Tags("github", "pr", "pull request")
 class OpenedPullRequest implements HandleEvent<PullRequest, PullRequest> {
     public handle(event: Match<PullRequest, PullRequest>): EventPlan {
-        const pr = event.root;
-
-        const cid = "pr_event/" + pr.repo.owner + "/" + pr.repo.name + "/" + pr.number;
-        const message = new LifecycleMessage(pr, cid);
-
-        message.addAction({
-            label: "Merge",
-            instruction: {
-                kind: "command",
-                name: "MergeGitHubPullRequest",
-                parameters: {
-                    issue: pr.number,
-                },
-            },
-        });
-
-        return EventPlan.ofMessage(message);
+        return new EventPlan();
```

Comment:
This was @cdupuis removing old lifecycle messages.  We might need empty handlers or old version of the same handler will fire.